src/components/LessonTitle/LessonTitle.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
+import theme from '../../styles/theme/theme';
+import { Label } from '../Label/Label';
+import { Grid } from '@mui/material';
+import { Typography } from '@mui/material';
+import { MainButton } from '../MainButton/MainButton';
+
+interface LessonTitleProps {
+  surahNameEn: string;
```

Comment:
Let's make sure we've got comments similar to what we have in other interfaces that have been defined.